src/ks/hangedman/Game.java

```diff
@@ -24,24 +30,40 @@ public void start() {
             if (!decision.equalsIgnoreCase("y")) {
                 break;
             }
-
         }
-
     }
 
     private List<String> readFile(String fileName) {
         List<String> wordsFromFile = new ArrayList<>();
         try (Scanner reader = new Scanner(Paths.get(fileName))) {
             while (reader.hasNextLine()) {
-                String row = reader.nextLine();
-                wordsFromFile.add(row);
+                String row = reader.nextLine().toLowerCase();
+                String[] singleWords = row.split("\\s+");
+                if (singleWords.length > 1) {
+                    wordsFromFile.clear();
+                    break;
+                } else if (singleWords.length < 1) {
+                    wordsFromFile.clear();
+                    break;
+                } else {
+                    if (isAlphabetic(singleWords[0])) {
+                        wordsFromFile.add(singleWords[0]);
+                    } else {
+                        wordsFromFile.clear();
+                        break;
```

Comment:
before each break, you could inform user, where is the problem, so he can better fix the file. If the file is large, it might be hard to localize, what is the problem, why you will not allow to use the list

Reply:
When reading of the file fails, the user receives info about it and also the problematic row is printed for easier localization. When there is an empty line, the program doesn't print the row, however - it won't be useful.
Also, I've added the line, where the program prints, that reading was successful and how many words it read, if that's the case.